src/web/pages.rkt

```diff
@@ -101,20 +89,18 @@
     (for/list ([point points])
       (for/list ([value point]) 
         (real->ordinal value repr))))
-        
+
   (define vars (test-vars test))
   (define bits (representation-total-bits repr))
   (define start-error (map ulps->bits-tenths start-errors))
-  (define target-error (and target-errors (map (lambda (alt-error) (map ulps->bits-tenths alt-error)) target-errors)))
+  (define target-error (map (lambda (alt-error) (map ulps->bits-tenths alt-error)) target-errors))
   (define end-error (map ulps->bits-tenths (car end-errors)))
 
   (define target-error-entries
-    (if (not target-error)
-        (cons "target" #f)
-        (for/list ([i (in-naturals)] [error-value (in-list target-error)])
-          (cons (format "target~a" (+ i 1)) error-value))))
+    (for/list ([i (in-naturals)] [error-value (in-list target-error)])
+        (cons (format "target~a" (+ i 1)) error-value)))
 
-  (define error-entries (append `(("start" . ,start-error) ("end" . ,end-error)) target-error-entries))
+  (define error-entries (list* `("start" . ,start-error) `("end" . ,end-error) target-error-entries))
```

Comment:
```suggestion
  (define error-entries (list* (cons "start" ,start-error) (cons "end" ,end-error) target-error-entries))
```

Reply:
Sorry I just used `list*` however it was used in the docs. This does look way more cleaner and I am guessing memory efficient.